### PR TITLE
feat: first take on the mobile activities

### PR DESCRIPTION
**Pull request by zacgodinez**

### CODE REVIEW

File: packages/ui/src/pages/accounts/components/layout/components/mobile/scrolling-buttons/index.tsx [hunk 1]
```diff
@@ -178,7 +163,8 @@ export const MobileScrollingButtons: React.FC<IProps> = ({ className }) => {
 							</Box>
 						}
 						rightIcon={
-							showActivities ? (
+							// eslint-disable-next-line no-constant-condition
+							false ? (
```
Comment (heathsnee) [inline packages/ui/src/pages/accounts/components/layout/components/mobile/scrolling-buttons/index.tsx:167]: why not remove this ?

Comment (zacgodinez) [reply packages/ui/src/pages/accounts/components/layout/components/mobile/scrolling-buttons/index.tsx:167]: we will bring it back when search is implemented?? we can remove tho

File: packages/ui/src/pages/accounts/components/layout/index.tsx [hunk 2]
```diff
@@ -11,7 +11,10 @@ import { useScroll } from 'ui/src/components/scroll-area-radix/use-scroll'
 import { ScrollPanel } from 'ui/src/components/scroll-panel'
 import * as panelViewStyles from 'ui/src/components/styles/panel-view-styles.css'
 import { useIsMobileWidth } from 'ui/src/hooks/use-is-mobile'
+import { useIsActivitiesVisible } from 'ui/src/pages/accounts/hooks/use-is-activities-visible'
 
+// import { ActivityList } from '../../components/activity-list/components/activity-list'
```
Comment (heathsnee) [inline packages/ui/src/pages/accounts/components/layout/index.tsx:16]: remove this comment ?

File: packages/ui/src/pages/accounts/components/resource-details/nft-collection.tsx [hunk 3]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
 import React from 'react'
 import { defineMessages, useIntl } from 'react-intl'
```
Comment (heathsnee) [inline packages/ui/src/pages/accounts/components/resource-details/nft-collection.tsx:3]: remove this eslint disable and remove intl ?

File: packages/ui/src/pages/accounts/components/resource-details/nft-item.tsx [hunk 4]
```diff
@@ -16,7 +17,8 @@ import { useNonFungibleData } from 'ui/src/hooks/dapp/use-entity-nft'
 import { getStringNftData } from 'ui/src/services/metadata'
 
 import FieldValue from './field-value'
-import * as styles from './styles.css'
+
+// import * as styles from './styles.css'
```
Comment (heathsnee) [inline packages/ui/src/pages/accounts/components/resource-details/nft-item.tsx:21]: lets delete styles if we dont use them, keep the bundle small

Comment (heathsnee) [reply packages/ui/src/pages/accounts/components/resource-details/nft-item.tsx:21]: style file as well

**Review by heathsnee — APPROVED** [review]
lgtm

**Pull request merged**